### ISSUE TITLE
Plugin stops working after an X period of time

### ISSUE THREAD

**Issue by imikeanaya** [MortJC/homebridge-platform-orbit#42]
I've been experiencing an issue where the Orbit system either thru the home app or by Siri does not respond to any turn on command. If initiated by the home app the switch just says "waiting..." and if triggered by Siri after a few seconds it responds that "the accessory is not present or responding" if I immediately switch to the Orbit app everything works without issues.
The only workaround I've found for this is restarting the homebridge system, unfortunately I have not measure for how long this connection is kept active and why it seems, cannot be established again.
I'm happy to provide more info or logs if needed, just need help on how to do so.

Thanks!

**Comment by MortJC** [comment]
Hi,

Can you confirm if you can control the system through Siri? I believe this
is currently broken, see
https://github.com/MortJC/homebridge-platform-orbit/issues/36.

If it's a different issue can you please send me the logs with debug
enabled?

**Comment by jdmtv001** [comment]
I cannot see the zones anymore in the homekit. Is all blank/white and also says scheduled but nothing is scheduled. I rebooted homebridge, rebooted Orbit, rebooted iPhone. it was working fine. I removed the plug-in and the bridge, reinstall. the same. 
<img width="503" alt="2022-03-30_18-06-03" src="https://user-images.githubusercontent.com/47341485/160945271-533fc679-8427-4047-a74c-dc5a484672ff.png">
